OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
+- akostadinov
+- pruan-rht
```

Comment:
@cjryan is working on upstream CI. So he is a better fit here.

Reply:
I was just adding people who already merge PRs here. Please either make your own file in a separate PR or merge this and edit it.